src/test/java/BoardTest.java

```diff
@@ -0,0 +1,121 @@
+            }
+        }
+    }
+
+    @Test
+    void testIsFull() {
+        Board board = new Board();
+        assertFalse(board.isFull()); // should be empty
+
+        // Fill board
+        for (int row = 0; row < 3; row++) {
+            for (int col = 0; col < 3; col++) {
+                String stringRow = String.valueOf(row);
+                String stringCol = String.valueOf(col);
+                board.placeX(new Position(Row.from(stringRow), Col.from(stringCol)));
+            }
+        }
+        assertTrue(board.isFull()); // should be full
+    }
+
+    @Test
+    void testIsOccupiedAt() {
+        Board board = new Board();
+        Position pos = new Position(topRow, leftCol);
+        assertFalse(board.isOccupiedAt(pos));
+
+        board.placeX(pos);
+        assertTrue(board.isOccupiedAt(pos));
+    }
+
+    @Test
+    void testPlaceXAndPlaceO() {
+        Board board = new Board();
+        Position posX = new Position(topRow, leftCol);
+        Position posO = new Position(midRow, midCOl);
+
+        board.placeX(posX);
+        board.placeO(posO);
+
+        assertTrue(board.isOccupiedAt(posX));
+        assertTrue(board.isOccupiedAt(posO));
+    }
+
+    @Test
+    void testGetStatusInProgress() {
+        Board board = new Board();
+        assertEquals(Board.Status.InProgress, board.getStatus());
+    }
+
+    @Test
+    void testGetStatusDraw() {
+
+        Board board = new Board();
+        board.placeX(new Position(topRow, leftCol));
+        board.placeO(new Position(topRow, midCOl));
+        board.placeX(new Position(topRow, rightCol));
+        board.placeO(new Position(midRow, leftCol));
+        board.placeX(new Position(midRow, midCOl));
+        board.placeO(new Position(midRow, rightCol));
+        board.placeX(new Position(botRow, leftCol));
+        board.placeO(new Position(botRow, midCOl));
+        board.placeX(new Position(botRow, rightCol));
+
+        assertEquals(Board.Status.Draw, board.getStatus());
+    }
+
+    @Test
+    void testGetStatusXWins() {
+        Board board = new Board();
+        board.placeX(new Position(topRow, leftCol));
+        board.placeX(new Position(topRow, midCOl));
+        board.placeX(new Position(topRow, rightCol));
+
+        assertEquals(Board.Status.XWins, board.getStatus());
+    }
+
+    @Test
+    void testToString() {
+        Board board = new Board();
+        board.placeX(new Position(topRow, leftCol));
+        board.placeO(new Position(midRow, midCOl));
+        String expected = "X..\n.O.\n...\n";
+        assertEquals(expected, board.toString());
+    }
+
+    @Test
+    void testSubtleBug() {
+        // test for diagonal bug
+        Board board = new Board();
+        board.placeX(new Position(topRow, leftCol));
+        board.placeX(new Position(midRow, midCOl));
+        board.placeX(new Position(botRow, rightCol));
+        assertEquals(Board.Status.XWins, board.getStatus());
+    }
+}
```

Comment:
The idea here was to write enough tests to catch the bug, but you've really only written one X wins test plus one that happens to catch the bug.  What about testing that both diagonal wins work? That all the row/col wins work? That both O and X can win? etc.